common/config/producer.go

```diff
@@ -13,10 +13,10 @@ type Producer struct {
 	Coinbase         string `json:"Coinbase"`
 	EntropyStorePath string `json:"EntropyStorePath"`
 
-	ExternalMiner bool `json:"externalMiner"`
-
 	coinbase types.Address
 	index    uint32
+
+	NoneSnapshotVerifier bool `json:"NoneSnapshotVerifier"`
```

Comment:
Maybe call it VirtualSnapshotVerifier?

Reply:
right

Reply:
https://github.com/vitelabs/go-vite/pull/607/commits/5dfa38ba7b1b328af582db9f8d1690ac6f536d03